__tests__/getCategories.test.js

```diff
@@ -37,4 +37,43 @@ describe("app ", () => {
         });
     });
   });
+  describe("/api/reviews", () => {
+    test("respond with a status of 200 and display an array of review objects sorted by date in descending order ", () => {
+      return request(app)
+        .get("/api/reviews")
+        .expect(200)
+        .then((reviews) => {
+          const dateOrderToTest = reviews.body.map((review) => {
+            return review.created_at.split("T")[0];
+          });
+
+          expect(dateOrderToTest).toBeSorted({ descending: true });
```

Comment:
instead of having to use a map here to format the dates and then see if it's in descending order - jest sorted lets us pass in what we want our data to be ordered by 
`expect([{ id: 3 }, { id: 2 }, { id: 1 }]).toBeSortedBy('id', {
  descending: true,
});`
This is covered in the jest docs, so here we can say we want our reviews to be ordered by `created_at`

Reply:
Thank you, Heather, on it